docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,186 @@
+[[semantic-search-semantic-text]]
+=== Tutorial: semantic search with `semantic_text`
+++++
+<titleabbrev>Semantic search with `semantic_text`</titleabbrev>
+++++
+
+beta[]
+
+This tutorial show you how to use the semantic text feature to perform semantic search on your data.
+
+Semantic text simplifies the {infer} workflow by providing {infer} at ingestion time and sensible default values automatically.
+You don't need to define certain settings and parameters or create {infer} ingest pipelines.
+
+The recommended way to use semantic search in the {stack} is following the semantic text workflow.
+When you need more control over the settings, you can still use the complete {infer} workflow (refer to  <<semantic-search-inference,this tutorial>> to review the process).
+
+This tutorial uses the `elser` service for demonstration but you can use any service and their supported models offered by the {infer-cap} API.
+
+
+[discrete]
+[[semantic-text-requirements]]
+==== Requirements
+
+To use `semantic_text` field type, you must have an {infer} endpoint deployed in
+your cluster using the <<put-inference-api>>.
+
+
+[discrete]
+[[semantic-text-infer-endpoint]]
+==== Create the {infer} endpoint
+
+Create an inference endpoint by using the <<put-inference-api>>:
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/sparse_embedding/my-elser-model <1>
```

Comment:
Suggest we rename this to `my-elser-endpoint` to avoid confusion about what's a model and what's an inference endpoint